test_module.py

```diff
@@ -3,6 +3,7 @@
 
 
 class UnitTests(unittest.TestCase):
+    maxDiff=None
```

Comment:
```suggestion
    maxDiff = None
```
Nitpick for style consistency.